### ISSUE TITLE
PSQL recreates a  graph that was already created with PG

### ISSUE THREAD

**Issue by audiBookning** [apache/incubator-age#67]
I encountered a strange situation when doing some experiments with AGE.

I used node-postgres (pg) to create a graph with 
```sql
SELECT create_graph('my_graph_name');
```
if i repeat the same query, i will correctly get an error
```sh
ERROR:  schema "my_graph_name" already exists
```
But if afterward i use psql to repeat the query, i get
```sh
NOTICE:  graph "my_graph_name" has been created
```
repeating it a 2º time in psql (or in pg) with gives the correct error.

I would expect to get an error, the first time, when trying the query in psql since i already created the graph in pg.

**Comment by JoshInnis** [comment]
It appears in the psql session you added a '2' to the graph name. Whereas in node-postgres you created a graph called 'my_graph_name'. Without seeing your scripts I cannot be sure, but it appears the reason why it was successful was because you used a different name.

**Comment by audiBookning** [comment]
I am sorry. The 2 is an artifact of copying the code to the issue.
I made another test with the 2 added to the end of the graph name to confirm the behaviour and when copying the code i forgot to remove it. 😪
I mean that the 2 in this case is not relevant to the issue.

I will edit the main issue to remove it.

**Comment by joefagan** [comment]
Please post your config to make sure we're pointing at the same server and same database when trying to recreating the graph.
.

**Comment by bitninejy** [comment]
@audiBookning I tried but I couldn't find this issues using psql.

Can you show your Development settings?

**Comment by audiBookning** [comment]
I am sorry for responding so late. Because of unforeseen reasons (irl), I have been out of the dev and "virtual world".

I have unfortunately no longer access to the environment  and code from witch this issue originated (related to the earlier mentioned accident). Its goal was to develop a quick prototype of a feature on another app.
I have tried to reconstruct it from memory, but wasn't able to re-confirm this issue. I did let out many things that i cannot remember in details so i cannot say any more about it.

I can only conclude that the issue is an isolated case and possibly even unrelated directly to AGE.
And worse is the fact that this is an issue that cannot be recreated.

This might be an non issue so i will close it. I will reopen it if i ever have more concrete factors about it.

I hope this issue hasn't taken to much time or energy of anyone.
